install/install.sh

```diff
@@ -80,11 +80,13 @@ else
        read -e -p "Invalid password, please retry: " -i "" db_user_password
     done
 fi
-supress_warning=`mysql_config_editor set --login-path=user_login --host=localhost --user=$db_user --password $db_user_password` >>  "$install_log" 2>&1
+#login with user account in mysql by veryfying mysql password
+mysql_config_editor set --login-path=mypath --host=localhost --user=$db_user --password ;
```

Comment:
Asking the user again for a password he possibly didn't create (see the section above) is asking for trouble.